language/spec/language_samples_spec.rb

```diff
@@ -45,20 +45,54 @@ def capture &block
     $stdout = real_stdout
   end
 
-  example "sentiment from text" do
+  example "document sentiment from text" do
     positive_text = "Happy love it. I am glad, pleased, and delighted."
     negative_text = "I hate it. I am mad, annoyed, and irritated."
 
     expect {
       sentiment_from_text project_id: @project_id, text_content: positive_text
-    }.to output(/^\d\.\d+ \(\d\.\d+\)/).to_stdout
+    }.to output(/Overall document sentiment: \(\d\.\d+\)$/).to_stdout
```

Comment:
@sgreenberg It was hard to see the difference between `\(\d\.\d+\)` and `\(-\d\.\d+\)` so it caused a little confusion

Vote: document the difference so it's easier to understand.  Inotherwords, go back to a well-named local variable, just like you had in the first place 😄 

Thought:

```ruby
positive_text   = "Happy love it. I am glad, pleased, and delighted."
negative_text   = "I hate it. I am mad, annoyed, and irritated."
positive_output = /\(\d\.\d+\)/
negative_output = /\(-\d\.\d+\)/

expect {
  # ...
```